### PR TITLE
Add bullet icon

### PR DESCRIPTION
**Pull request by Malinicz**
Add bullet icon to web icons set:

image

image

### CODE REVIEW

File: src/images/icons/bullet.svg [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512">
```
Comment (Malinicz) [inline src/images/icons/bullet.svg:1]: went through svgomg

**Review by bartoszlorek — APPROVED** [review]

File: src/images/icons/bullet.svg [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+  <title>bullet</title>
+  <g fill="none" fill-rule="evenodd">
```
Comment (coderitual) [inline src/images/icons/bullet.svg:3]: can we remove this g entirely?

**Review by coderitual — APPROVED** [review]

**Comment by coderitual** [comment]
@Malinicz Could you please add icon screenshot from storybook/style guide for release purposes?

File: src/images/icons/bullet.svg (hunk 2 above)
Comment (coderitual) [reply src/images/icons/bullet.svg:3]: @Malinicz ignore that comment, looks we need it

Comment (coderitual) [reply src/images/icons/bullet.svg:3]: @Malinicz alternatively you can remove group along with path. Circle alone is enough

**Comment by matzimowski** [comment]
@Malinicz could u please add a screenshot? 🍭

**Review by matzimowski — APPROVED** [review]

File: src/images/icons/bullet.svg [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+    <path d="M0 0h512v512H0z"/>
+    <circle fill="#000" cx="256" cy="256" r="64"/>
```
Comment (coderitual) [inline src/images/icons/bullet.svg:5]: we need to remove fill color for web icons

**Review by coderitual — CHANGES_REQUESTED** [review]
@Malinicz To make color overrides working properly, resolving commented issues should do the job (removing fill from cricle, remove group and remove path)
image

File: src/images/icons/bullet.svg (hunk 2 above)
Comment (Malinicz) [reply src/images/icons/bullet.svg:3]: @coderitual I wanted to set fill as `currentColor`, but then I checked other icons and most of them have fill set to `#000`. However, I left below and it seems to work fine:

```
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512">
  <title>bullet</title>
  <circle cx="256" cy="256" r="64" fill="currentColor" fill-rule="evenodd"/>
</svg>
```

Comment (coderitual) [reply src/images/icons/bullet.svg:3]: @Malinicz could you give me an example of web icon having fill set to #000? if there is any it's a bug

**Review by coderitual — APPROVED** [review]
❤️  thanks!

I confirm there are other icons with fill color set to #000. This is of course a bug so you might have been tricked because of them.

**Pull request merged**